docs/_posts/2018-01-01-SpaceVim-Newsletter-Never-lost-never-give-up.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Newsletter #2 - Never lost, Never give up
+categories: newsletter
+excerpt: "We know exactly what our purpose is, and we keep trying to do it, never get lost, never give up..."
+---
+
+# [newsletter](https://spacevim.org/development#newsletter) > Never lost, Never give up
+
+Welcome to the second newsletter for SpaceVim, a project that hopes to turn vim to be an modular IDE for most languages.
+
+## What is SpaceVim
+
+SpaceVim is a distribution of the vim editor that's inspired by [spacemacs](https://github.com/syl20bnr/spacemacs). It manages collections of plugins in layers, which help collect related packages together to provide features. For example, the [python layer](http://spacevim.org/layers/lang/python/) collects [deoplete.nvim](https://github.com/Shougo/deoplete.nvim/), [neomake](https://github.com/neomake/neomake) and [jedi-vim](https://github.com/davidhalter/jedi-vim) together to provides autocompletion, syntax checking, and documentation lookup. This approach helps keep configuration organized and reduces overhead for the user by keeping them from having to think about what packages to install.
```

Comment:
"SpaceVim is a distribution of the vim editor that's inspired by" => "SpaceVim is a vim distribution which's inspired by",
"collections" => "collection",
"the user" => "users"